### ISSUE TITLE
Updating legacy code for controls

### ISSUE THREAD

**Issue by dyablohunter** [openseadragon/openseadragon#1804]
Hey guys,

I was looking over the library and I think the project would benefit greatly from updating the controls part of it.
It is written in a style that can now be improved greatly, by just using some pure CSS and SVG.
A big part of the code will be eliminated while achieving the same result with better performance.
We don't actually need the fancy 4 state by default, but if you all agree that we do, it can be done by simply changing the SVG's with some CSS, changing fill, size, position, opacity, animation, anything.
I already have the free icons for the most important 5 of them and I will provide the code according to the solutions voted.
zoom_in/zoom_out/reload/fullscreen/info

Let's choose either one simple state for the control buttons, or 3-4 states but done through CSS states, not all that JS currently used.

**Comment by iangilman** [comment]
Indeed, our button design is terribly out of date, both visually and technologically.

If you're up for it, I'd love to explore what can be done.

Thoughts:

* We'll need to provide the full set of buttons, of course.
* I like the fact that we have a hover effect, and the fact that the controls fade away when not in use. Those are the states I care about.
* Whatever we do should still support custom buttons (since many sites already do that). For that reason, I imagine we won't be able to rip out the old code. We'll simply be providing a new alternative.
